lib/main.ts

```diff
@@ -92,13 +97,16 @@ function symbolCompareFunction(a: ts.Symbol, b: ts.Symbol) {
 }
 
 /**
- * Traverses the node tree to construct the text without comments.
+ * Traverses the node tree to construct the text without comments and privates.
  */
-function getTextWithoutComments(node: ts.Node): string {
-  // TODO: test more
+function getSanitizedText(node: ts.Node): string {
+  if (node.flags & ts.NodeFlags.Private) {
```

Comment:
interesting. this means that we no longer see default values being assigned
- this is sad but there isn't much we can do about that.


On Mon, Jun 20, 2016 at 8:56 AM Jason Choi <notifications@github.com> wrote:

> In lib/main.ts
> <https://github.com/angular/ts-api-guardian/pull/6#discussion_r67715153>:
>
> >   */
> > -function getTextWithoutComments(node: ts.Node): string {
> > -  // TODO: test more
> > +function getSanitizedText(node: ts.Node): string {
> > +  if (node.flags & ts.NodeFlags.Private) {
>
> if you mean constructor(private a: string), I believe it is always
> converted to
>
> private a;
> constructor(a: string)
>
> by the d.ts generator.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/ts-api-guardian/pull/6/files/6dc6beb0b16b8a8af79383784bbe54779104e8ce..ae7ed528eba57cea36e418da85cd2e303f73b2f6#r67715153>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANM6MInqdnh0jhlPN-5XDidvEN92cccks5qNrgSgaJpZM4I40lA>
> .
>